internal/core/config.go

```diff
@@ -47,6 +47,7 @@ type ClusterConfiguration struct {
 	ResourceBehaviors        []ResourceBehavior                `yaml:"resource_behavior"`
 	RateBehaviors            []RateBehavior                    `yaml:"rate_behavior"`
 	QuotaDistributionConfigs []*QuotaDistributionConfiguration `yaml:"quota_distribution_configs"`
+	MailForms                MailForms                         `yaml:"mail_templates"`
```

Comment:
Variable names should match config file field names if possible:
```suggestion
	MailTemplates            MailTemplateConfiguration         `yaml:"mail_templates"`
```